twoliter/embedded/Makefile.toml

```diff
@@ -1073,6 +1073,60 @@ pubsys \
 '''
 ]
 
+[tasks.fetch-variant]
+dependencies = ["publish-setup-without-key"]
+script_runner = "bash"
+script = [
+'''
+set -e
+
+export PATH="${TWOLITER_TOOLS_DIR}:${PATH}"
+
+pubsys \
+   --log-level "${PUBLISH_LOG_LEVEL}" \
+   --infra-config-path "${PUBLISH_INFRA_CONFIG_PATH}" \
+   \
+   fetch-variant \
+   \
+   --repo "${PUBLISH_REPO}" \
+   --arch "${BUILDSYS_ARCH}" \
+   --variant "${BUILDSYS_VARIANT}" \
+   \
+   --variant-manifest "${BUILDSYS_ROOT_DIR}/variants/${BUILDSYS_VARIANT}/Cargo.toml" \
+   --filename-prefix "${BUILDSYS_NAME_FULL}" \
+   --root-role-path "${PUBLISH_REPO_ROOT_JSON}" \
+   \
+   --outdir "${BUILDSYS_VARIANT_DIR}"\
+'''
+]
+
+[tasks.fetch-ova]
```

Comment:
This could also be `fetch-friendly`, but I imagine `fetch-ova` is more intuitive.